openfecli/commands/input.yaml

```diff
@@ -0,0 +1,43 @@
+# things behind a "method" key have to match up to a known thing in openfe
+
+network:
+  method: plan_radial_network
+  scorer: default_lomap_score
+
+# Question:
+# use settings here, to match Protocol
```

Comment:
I honestly don't remember which approach was easier for a JSON Schema. I think the not nested version is a little easier: you define a single subschema with a literal for `method` and the appropriate list of settings keys. With nested settings, I think you have to define the subschema with the `method` literal and point another subschema for the settings. That would make documentation, e.g., as generated by `sphinx-jsonschema`, a little ugly. TBH, it is really just a question of which is easier with whatever tool we'll do to generate the JSON schema (do *not* go down the road of rolling your own).